src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleCloudStorageDAO.scala

```diff
@@ -307,6 +351,7 @@ class HttpGoogleCloudStorageDAO(
       .build()
   }
 
+  def adminGroupName = s"${groupsPrefix}-ADMIN@${appsDomain}"
```

Comment:
doug and i discussed this, and decided that we can use the groupsPrefix for this purpose